internal/controllers/build_sign_events_reconciler.go

```diff
@@ -97,27 +100,18 @@ func (r *JobEventReconciler) Reconcile(ctx context.Context, req reconcile.Reques
 
 	kernelVersion := pod.Labels[constants.TargetKernelTarget]
 
-	mod := kmmv1beta1.Module{}
-
-	moduleNSN := types.NamespacedName{
-		Namespace: req.Namespace,
-		Name:      pod.Labels[constants.ModuleNameLabel],
+	if nor := len(pod.OwnerReferences); nor != 1 {
```

Comment:
Why?

Reply:
It is always to better use interface then fields (IMHO). Interface implementation can change , and we will be able to enjoy those changes ( additional verifications, whatever).  Also we are not dependent on the structure itself, in case it changes

Reply:
I think using the interface implementation would make that line slightly less readable, and would make little sense here: we do know that `pod` is a `Pod` struct. I would agree with you if we were to pass that object to another function, but that is not the case here.

Reply:
i think readability is exactly the same: the function name is very clear. Your code will work as of now, but what if, in the future the field will be changed? The API won't change (at least the the chances of it changing are much less ), so our code is covered